<launch>
  <!-- This launch file starts everything necessary for object recognition. -->

  <node pkg="rail_recognition" name="pc_recognition" type="pc_recognition" output="screen" />

</launch>
